web/src/api/ApiClient.ts

```diff
@@ -51,6 +51,33 @@ export class ApiClient {
             throw err
         }
     }
+
+    async register(
+        email: string,
+        password: string,
+        age: number,
+        gender: string
+    ) {
+        try {
+            return await this.baseRequest<void>(`user`, {
+                method: 'POST',
+                body: JSON.stringify({
+                    email,
+                    password,
+                    age,
+                    gender,
+                }),
```

Comment:
"banned" field is also required according to spec. but actually it doesn't make much sense to specify a "banned" field during registration. we should discuss these inconsistencies with the other groups to ensure compatibility between modules from different groups.

Reply:
For now ill just send `banned: false` to adhere to the spec